procrastinate/cli.py

```diff
@@ -38,7 +40,7 @@ def click_set_verbosity(ctx: click.Context, param: click.Parameter, value: int)
 def set_verbosity(verbosity: int) -> None:
     level = get_log_level(verbosity=verbosity)
     logging.basicConfig(level=level)
-    level_name = logging.getLevelName(level)
+    level_name = logging.getLevelName(level, format=LOG_FORMAT)
```

Comment:
[`getLevelName`](https://docs.python.org/3/library/logging.html#logging.getLevelName) has no argument `format`. Did you mean to add this to basicConfig instead ?

Reply:
Yes! 🤦‍♂️ Good catch. I copy/paste only the arguments from other project. I'll fixit later